tests/unit/models/test_theme.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from consultation_analyser import factories
+from consultation_analyser.consultations import models
+
+
+def set_up_frameworks():
+    question_part = factories.QuestionPartFactory(type=models.QuestionPart.QuestionType.FREE_TEXT)
+
+    # Set up initial framework - replicate a run of the theme generation task.
+    theme_generation_run = factories.ExecutionRunFactory(
+        type=models.ExecutionRun.TaskType.THEME_GENERATION
+    )
+    framework_1 = factories.FrameworkFactory(
+        execution_run=theme_generation_run, question_part=question_part
+    )
+    factories.ThemeFactory(name="X", framework=framework_1)
+    factories.ThemeFactory(name="Y", framework=framework_1)
+    factories.ThemeFactory(name="Z", framework=framework_1)
+
+    # Create a new framework amending these themes
+    framework_2 = framework_1.amend_framework(
+        user=factories.UserFactory(), change_reason="I wanted to change the themes."
+    )
+    return framework_1, framework_2
+
+
+@pytest.mark.django_db
+def test_amend_theme():
+    framework_1, framework_2 = set_up_frameworks()
+    theme_x = models.Theme.objects.get(name="X", framework=framework_1)
+    models.Theme.objects.get(name="Y", framework=framework_1)
+
+    with pytest.raises(ValueError):
```

Comment:
Is it worth asserting the error message? If nothing else, it might make it clearer what the problem is when reading the test.

Reply:
Updated: https://github.com/i-dot-ai/consult/blob/77148101c0e62e4d5cdf29d2cbf8b0401aa2c830/tests/unit/models/test_theme.py#L39